modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java

```diff
@@ -448,17 +448,15 @@ public String toVarName(String name) {
         if (name.matches("^[A-Z_]*$")) {
             name = name.toLowerCase();
         }
-
-        // camelize (lower first character) the variable name
-        // petId => pet_id
-        name = underscore(name);
```

Comment:
don't need `underscore` anymore?

Reply:
Not since we introduced modelPropertyNaming which sets the variable name to camelCase.